uint8 imgType
---
uint8 status
